creatures-hunting-service/src/test/java/cz/fi/muni/pa165/service/facade/WeaponFacadeUnitTest.java

```diff
@@ -0,0 +1,231 @@
+package cz.fi.muni.pa165.service.facade;
+
+import cz.fi.muni.pa165.dto.MonsterDTO;
+import cz.fi.muni.pa165.dto.WeaponCreateDTO;
+import cz.fi.muni.pa165.dto.WeaponDTO;
+import cz.fi.muni.pa165.entity.Monster;
+import cz.fi.muni.pa165.entity.Weapon;
+import cz.fi.muni.pa165.enums.MonsterAgility;
+import cz.fi.muni.pa165.enums.WeaponType;
+import cz.fi.muni.pa165.facade.WeaponFacade;
+import cz.fi.muni.pa165.service.BeanMappingService;
+import cz.fi.muni.pa165.service.MonsterService;
+import cz.fi.muni.pa165.service.WeaponService;
+import cz.fi.muni.pa165.service.config.ServiceConfiguration;
+import org.mockito.InjectMocks;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * Implementation of Facade Unit tests
+ *
+ * @author Pavel Vyskocil <vyskocilpavel@muni.cz>
+ */
+@ContextConfiguration(classes = ServiceConfiguration.class)
```

Comment:
Since you are mocking the beanMappingService, you don't need to use this configuration. Also please remove the 'extedns AbstractTestNgSpringContextTests' part.